src/IssueViz.Security/Hotspots/HotspotsList/ViewModels/HotspotsControlViewModel.cs

```diff
@@ -66,10 +65,10 @@ internal sealed class HotspotsControlViewModel : IHotspotsControlViewModel, INot
             IIssueSelectionService selectionService,
             IThreadHandling threadHandling)
         {
+            this.threadHandling = threadHandling;
             AllowMultiThreadedAccessToHotspotsList();
 
             this.selectionService = selectionService;
-            this.threadHandling = threadHandling;
```

Comment:
This view model already had an instance of `IThreadHandling`, but still had a call to `ThreadHelper`.

I moved the `this.threadHandling = threadHandling;` call up to line 68 since it is used by `AllowMultiThreadedAccessToHotspotsList()` uses it now.